build.psm1

```diff
@@ -681,13 +681,13 @@ function Publish-PSTestTools {
 
     Find-Dotnet
 
-    $tools = @("$PSScriptRoot/test/tools/EchoArgs", "echoargs"), @("$PSScriptRoot/test/tools/CreateChildProcess", "createchildprocess")
+    $tools = ,@("${PSScriptRoot}/test/tools/TestExe", "testexe")
```

Comment:
I was trying to follow the existing convention for EchoArgs and CreateChildProcess.  Since DotNet calls it an exe, I think TestExe is appropriate.  "TestRunner" to me means something that runs tests.